### DISCUSSION TITLE
Error! request to https://api.zeit.co/v1/frameworks failed, reason: connect ETIMEDOUT 13.52.46.156:443

### DISCUSSION THREAD

**Discussion by Vadorequest** [vercel/next.js#12506] [Help]
```sh
yarn start
yarn run v1.22.0
$ now dev --listen 8888
> UPDATE AVAILABLE Run `yarn add now@latest` to install Now CLI 18.0.0
> Changelog: https://github.com/zeit/now/releases/tag/now@18.0.0
Now CLI 17.1.1 dev (beta) — https://zeit.co/feedback
⠼ Retrieving project…Error! request to https://api.zeit.co/v1/frameworks failed, reason: connect ETIMEDOUT 13.52.46.156:443
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

```

I cannot start my project locally du to this error. Was working fine 20mn ago.

**Comment by Vadorequest** [comment]
Issue is fixed (project starts again locally), even though it's still a bit slower than usual.

**Comment by marwajomaa** [comment]
Hi, @Vadorequest How did you solve this issue, I am facing the same problem, not able to start next.js project due to this error?!!

**Reply by Vadorequest** [reply]
I don't quite remember exactly, but I believe it was an issue from Vercel vendor and it got fixed by them. Maybe it'll will work again in 20mn?
